cmd/delete-all.go

```diff
@@ -66,15 +120,89 @@ var DeleteAll = &cobra.Command{
 	},
 }
 
-func getPage(resourceName string) ([]string, error) {
-	resp, err := getResource([]string{resourceName, "page[limit]", "25"})
+//
+func getParentIds(ctx context.Context, resource resources.Resource) ([][]string, error) {
```

Comment:
Probably can refactor this, so that instead of taking a resource, and getting all parents, let's just get the caller to pass the initial value in. It's kind of weird the way it is written, and hard to reason about.